src/helpers/misc.ts

```diff
@@ -124,10 +125,13 @@ export function computeTextLinesHeight(textLineHeight: number, numberOfLines: nu
 /**
  * Get the default height of the cell given its style.
  */
-export function getDefaultCellHeight(style: Style | undefined): Pixel {
-  // TO DO: take multi text line into account to compute the real cell height in case of wrapping cell
-  const fontSize = computeTextFontSizeInPixels(style);
-  return computeTextLinesHeight(fontSize) + 2 * PADDING_AUTORESIZE_VERTICAL;
+export function getDefaultCellHeight(cell: Cell | undefined): Pixel {
+  if (!cell || !cell.content) {
+    return DEFAULT_CELL_HEIGHT;
+  }
+  const fontSize = computeTextFontSizeInPixels(cell.style);
+  const multiLineText = cell.content.split("\n");
```

Comment:
```suggestion
  const multiLineText = cell.content.split(NEWLINE);
```
we have a NEWLINE constant somewhere